sdk/eventgrid/azure-eventgrid/azure/eventgrid/_helpers.py

```diff
@@ -38,6 +37,7 @@ def generate_sas(endpoint, shared_access_key, expiration_date_utc, **kwargs):
     :keyword str api_version: The API Version to include in the signature.
      If not provided, the default API version will be used.
     :rtype: str
+    :return: A shared access signature string.
```

Comment:
same as Kashif pointed out here :) 
https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html#the-sphinx-docstring-format